### DISCUSSION TITLE
External Links in Jupyter Notebooks Not Opening in New Tabs

### DISCUSSION THREAD

**Discussion by mscherrmann** [alshedivat/al-folio#2229] [Q&A] [answered]
I'm using the al-folio theme for my GitHub Pages website, which includes a blog section. Some of my blog posts are Jupyter notebooks that have been converted to HTML. I've noticed that external links within these notebooks do not open in new tabs when clicked, but instead are opened directly within my webpage. I would like these links to open in new tabs to improve user experience. Has anyone an idea how to do that? See for example here:
http://moritzscherrmann.com/blog/2024/GPU-Memory_BERT/

Thank you very much!

**Comment by george-gca** [comment, marked as answer]
For this you'll have to modify all the links inside the converted notebook to include `target="_blank"`. You can base this code on the [code responsible for changing the jupyter notebook theme](https://github.com/alshedivat/al-folio/blob/b607e580adbf2b96194dce79146f8aa201016da3/assets/js/theme.js#L49) to dark/light.

In that code we retrieve the `body` element from the notebook html code. From that point on you should be able to retrieve all the `<a>` elements and modify them. If you manage to solve this, would you mind sending a PR?

**Reply by mscherrmann** [reply]
Perfect, thank you for your quick response. It works for me now. I created the PR.
